homeassistant/components/camera/xiaomi.py

```diff
@@ -36,7 +36,7 @@
     vol.Required(CONF_MODEL): vol.Any(MODEL_YI,
                                       MODEL_XIAOFANG),
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.string,
-    vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
+    vol.Required(CONF_PATH, default=DEFAULT_PATH): cv.string,
```

Comment:
Making this required, should remove the default.
It would make no sense to keep it.